pulsar/README.md

```diff
@@ -0,0 +1,90 @@
+# Pulsar Bindings
+This document defines how to describe Apache Pulsar specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+## Server Binding Object
+
+This object contains information about the server representation in Pulsar.
+
+##### Fixed Fields
+
+Field Name | Type | Description |
+---|:---:|:---:|
+`tenant` | String | **Optional**. The pulsar tenant. If omitted, "public" must be assumed. |
+`bindingVersion` | String | **Optional**. The version of this binding. If omitted, "latest" MUST be assumed. |
+
+This object MUST contain only the properties defined above.
+
+##### Example
+
+```yaml
+servers:
+  production:
+    bindings:
+      pulsar:
+        tenant: contoso
+        bindingVersion: '0.1.0'
+```
+
+<a name="channel"></a>
+## Channel Binding Object
+This object contains information about the channel representation in Pulsar
+
+##### Fixed Fields
+
+Field Name | Type | Description |
+---|:---:|:---:|
+`namespace` | String | **Optional**. The namespace, the channel is associated with. If omitted, "public" MUST be assumed. |
```

Comment:
```suggestion
`namespace` | String | **Optional**. The namespace the channel is associated with. If omitted, "public" MUST be assumed. |
```

Reply:
Caught a copy paste mistake, with the defaulted namespace - this has been rectified, as this is also required.